backend/apps/authentication/models.py

```diff
@@ -0,0 +1,79 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    email = models.EmailField(unique=True, verbose_name="Email address")
+    is_verified = models.BooleanField(default=False, verbose_name="Email verified")
+
+    USERNAME_FIELD = 'email'
+    REQUIRED_FIELDS = []
+
+    username = models.CharField(max_length=150, unique=True, blank=True, null=True)
+
+    class Meta:
+        verbose_name = "User"
+        verbose_name_plural = "Users"
+
+    def __str__(self):
+        return self.email
+
+    def save(self, *args, **kwargs):
+        if not self.username:
+            self.username = self.email
+        super().save(*args, **kwargs)
+
+
+class StartupProfile(models.Model):
+    user = models.OneToOneField(
+        User,
+        on_delete=models.CASCADE,
+        related_name='startup_profile'
+    )
+    company_name = models.CharField(max_length=255, verbose_name="Company name")
+    short_pitch = models.TextField(blank=True, verbose_name="Short pitch")
+    website = models.URLField(blank=True, verbose_name="Website")
+    phone = models.CharField(max_length=20, blank=True, verbose_name="Contact phone")
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        verbose_name = "Startup Profile"
+        verbose_name_plural = "Startup Profiles"
+
+    def __str__(self):
+        return f"{self.company_name} ({self.user.email})"
+
```

Comment:
If apps.startups has its own StartupProfile model, this may cause a naming conflict - check to make sure there are no duplicates.
(Your previous files from apps/startups/models.py and apps/authentication/models.py contain two different StartupProfiles - you need to merge or rename one of them.)